src/components/Chips/types.ts

```diff
@@ -0,0 +1,22 @@
+import { IconPropTypes } from '../Icon/types'
+
+export type TChipsColors =
+  | 'primary'
+  | 'disabled'
+  | 'brand'
+  | 'danger'
+  | 'warning'
+  | 'success'
+  | 'information'
+  | 'discovery'
+export interface TChipsProps extends IFormCompProps {
+  text?: string | number
```

Comment:
I think text is a mandatory field, and type is string or ReactElement (not number)